### PR TITLE
Save log for different runs of smoke tests

### PR DESCRIPTION
**Pull request by daniele-moro**

### CODE REVIEW

**Comment by daniele-moro** [comment]
retest this please

**Comment by daniele-moro** [comment]
retest this please

**Comment by daniele-moro** [comment]
retest this please

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -187,14 +187,15 @@ void runSTCSmokeTest(String topo) {
 
 void exportArtifactsScenarios(String topo) {
   dir("${env.WORKSPACE}/up4/scenarios") {
+    timestamp = sh(script: "echo `date +%s`", returnStdout: true).trim()
```
Comment (charlesmcchan) [inline .jenkins/Jenkinsfile:190]: Is it better to use a more human readable format? For example, `+%Y%m%d-%H%M%S`

**Review by pierventre — APPROVED** [review]
Is it not easier to use the `retryCount` ?

**Comment by daniele-moro** [comment]
> Is it not easier to use the `retryCount` ?

I tried that approach (as you may see on previous commit), however, I wasn't able to correctly increase, reset and use the retryCount variable in the step.

**Comment by pierventre** [comment]
> > Is it not easier to use the `retryCount` ?
> 
> I tried that approach (as you may see on previous commit), however, I wasn't able to correctly increase, reset and use the retryCount variable in the step.

mm ok - I dont have any idea atm

**Review by ccascone — APPROVED** [review]

**Pull request merged**